docker/gcp_pudl_etl.sh

```diff
@@ -89,6 +108,8 @@ run_pudl_etl 2>&1 | tee $LOGFILE
 # Notify slack if the etl succeeded.
 if [[ ${PIPESTATUS[0]} == 0 ]]; then
     notify_slack "success"
+    echo $COMMIT_HASH > commit_hash.txt
+    echo $SHA256_HASH > sha256_hash.txt
```

Comment:
This is the only thing that really needs to happen in the script -- once the build is complete, record both of these hashes and make sure they get distributed with the other build outputs.

Because there are several databases being produced, the name of the file recording the hash of the database should indicate which database it applies to, and while we're doing this we might as well produce hashes for all of the SQLite DBs and the big single-file EPA CEMS Parquet output.

Reply:
the commit hash of the current Git branch to a text file named commit_hash.txt and the SHA256 hash of the PUDL SQLite output to a text file named sha256_hash.txt.